context-free.md

```diff
@@ -2,21 +2,31 @@
 
 ##  Who is this programming language for?
 
+The programming language is for people that want to be able to create digital art without learning how to render it in a different programming language. In ContextFree, the only code that the user needs to write involves the image that they produce. The user does not need to learn about graphics libraries or additional features of a more complex language. In general, ContextFree is optimal for someone who wants to create graphical images who has a basic understanding of computer science.
 
 ## What is easy to do in this language? Why is it easy?
 
+It is easy to produce simple shapes in the language because it includes `CIRCLE` and `SQUARE` instructions. In order to produce a simple unit circle, the user only needs to write one rule and does not need to specify any adjustments to the shape. Similarly, it is easy to color the background and size the canvas because of the supported methods in ContextFree. The language makes it intuitive to reposition elements with use of `x` and `y` transformations.
 
 ## What is hard to do in this language? Why is it hard?
 
+Without a background in computer science, this language would make it difficult to repeat objects. It is simple to implement recursion (call a rule within itself), but more difficult to understand how to use it. The language includes virtually no debugging and limited error messages. Given its simplicity, this is understandable. As mentioned above, the language seems to be optimized for someone with a basic understanding of computer science who wants to create complex graphics. 
+
+It is also hard to make functions that accept arguments. The documentation includes sections about this but does not include extensive examples. The process of creating a function to rotate and transform a shape (passed to the function as a parameter) is not intuitive or simple in ContextFree.
```

Comment:
So when you compare typical images that are made with ContextFree, how do they differ from images made in other language/programs? While the actual programming aspect is important, it is also vital to consider the actual results that want to be made with the language. Like are there easier and harder types of images you can make in ContextFree? Does that differ from other software that allows for image creation?